data/human/human missions.txt

```diff
@@ -7453,3 +7453,1280 @@ mission "Adelita 5"
+			`	His voice becomes somber, "Perhaps too good. I started to forget where the genunine Tavish began, and the counterfeit Tavish ended. So, I stopped hiding who I was and made my interests and studies known to all!"`
+			`	A wry grin spreads across his face, "And was promptly thrown out! So, now I am here with you and Kormak following my passion, on the forefront of discovery and adventure! And I am not too disconcerted that my old talents have some use."`
+			`	After you arrive back at your ship, Tavish says, "Now we shall see a man in Hephaestus about some misplaced equipment," and hops aboard.`
+				accept
+		
+			label kormak1
+				action "kormak1"
+			`	Kormak cracks a fist and steps right up to the bar. With deft movement Kormak has the barkeeper's neck in his veiny hands.`
+			`	"Information." Kormak says.`
+			`	Despite the bartender's alarming situation, he appears unphased. He gasps a little but says nothing else. It's almost as if he's waiting...`
+			`	Just then three burly men plow into Kormak's side. Kormak drops the bartender as he crashes sideways into an occupied booth. This prompts more angry pirates to enter the fray as an all out bar brawl breaks out.`
+			`	Kormak is big, fast, and furious, and seems to have no qualms doing lasting damage. Fingers snap, eyes are gouged, arms snap, and blood splatters everywhere. Unfortunately the pirates seem to have an endless supply of bodies to throw at Kormak and eventually he's worn down enough for them to throw him out of the bar.`
+			`	With him gone the fighting dies down. Eventually the bartender gathers himself up well enough to look you in the eye and say, "Now it's 100,000 credits for that information."`
+				goto options
+	
+			label tav2
+			action
+				clear "kormak1"
+			`	Tavish steps around the broken bar bottles and stools and gives the bartender his winningest smile.`
+			`	"My good man, you seem like a discerning fellow. Always looking to make a deal with an eye out for profit, am I right?"`
+			`	The bartender looks piqued and slowly nods, "Aye."`
+			`	"You're probably wondering what a pompous overdressed man like me is doing in your quaint out-of-the-way boutique. Well, I was looking for just the sort of enterprising soul who knows a thing or two about making a shilling."`
+			`	Tavish proceeds to describe a long and complex scam utilizing unwitting pirates, corrupt Syndicate trade officials, and fabulously wealthy marks from the paradise world. As he talks the barkeep leans further and further in, his eyes widening beneath a growing devious grin."`
+			`	Finally, Tavish concludes with, "We're kindred spirits, you and I, and that's why I know this will work. So what do you say? 25,000 credit buy in, plus the information on our first unwitting mark, and you're all set to be rich."`
```

Comment:
```suggestion
			`	Finally, Tavish concludes with, "We're kindred spirits, you and I, and that's why I know this will work. So what do you say? 25,000 credit buy-in, plus the information on our first unwitting mark, and you're all set to be rich."`
```